supply-chain/catalog/trivy-scanning.hbs.md

```diff
@@ -0,0 +1,64 @@
+# Trivy Scanning Component
+
+## Description
+
+This document describes the Trivy Scanning Component for Tanzu Supply Chains. The Trivy Component provides out of the box trivy scanning that invokes [Supply Chain Security Tools - Scan 2.0](../../scst-scan/scan-2-0.hbs.md) in a Tanzu Supply Chain. See aquasecurity trivy [docs](https://github.com/aquasecurity/trivy) for more details on the trivy scanner.
+
+## API
+
+Component input: `image` is an input from the buildpack component
+
+Workload input: `spec.registry` is where the scan results are uploaded
+
+Workload input: `spec.source` is for the source component to git clone
+
+Workload input: `spec.scanning` is for configurations for Scan 2.0
+
+Configuring the Trivy Scanning component:
+```
+spec:
+  registry:
+    repository: REGISTRY-REPOSITORY
+    server: REGISTRY-SERVER
+  source:
+    git:
+      branch: BRANCH-NAME
+      url: GIT-URL
+  scanning:
+    service-account-scanner: SERVICE-ACCOUNT-SCANNER
+    service-account-publisher: SERVICE-ACCOUNT-PUBLISHER
+    active-keychains:
+    workspace:
+      bindings: WORKSPACE-BINDINGS
+      size: WORKSPACE-SIZE
+```
+Where:
+* REGISTRY-REPOSITORY is the registry server repository where scan results are uploaded
+* REGISTRY-SERVER is the registry server where scan results are uploaded
+* BRANCH-NAME is the git branch ref
+* GIT-URL is the url to the git source repository
+* SERVICE-ACCOUNT-SCANNER is the service account that runs the scan. It must have read access to image.
+* SERVICE-ACCOUNT-PUBLISHER is the service account that uploads results. It must have write access to scanResults.location.
+* spec.scanning.active-keychains is an array of enabled credential helpers to authenticate against registries using workload identity mechanisms.
+* WORKSPACE-BINDINGS are additional array of secrets, ConfigMaps, or EmptyDir volumes to mount to the running scan. The name is used as the mount path.
+* WORKSPACE-SIZE is size of the PersistentVolumeClaim the scan uses to download the image and vulnerability database.
+
+**Note:** See more examples of these IVS configurations [here](../../scst-scan/ivs-create-your-own.hbs.md).
+
+## Dependencies
+
+* To enable this Trivy Scanning Component in the Tanzu Supply Chain workload, the following packages must be installed:
```

Comment:
The other teams named this using their general name but I felt this may be more accurate cause for example there's a supplychain v1 and supplychain v2 packages that get confusing due to the name.